introduction.md

```diff
@@ -0,0 +1,7 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed bibendum faucibus faucibus. Morbi consequat sem magna, ut dignissim metus efficitur nec. Nullam viverra odio at turpis tincidunt, sit amet faucibus enim ultrices. Nullam rhoncus gravida semper. Maecenas erat risus, aliquet non pellentesque a, auctor sit amet ligula. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In hac habitasse platea dictumst. Cras laoreet leo quam, in pulvinar felis tincidunt ac. Sed faucibus felis quis viverra dapibus.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.